pkg/gardenlet/controller/managedseed/valueshelper_test.go

```diff
@@ -105,8 +105,7 @@ var _ = Describe("ValuesHelper", func() {
 				},
 			},
 			FeatureGates: map[string]bool{
-				string(features.ReversedVPN): true,
-				string(features.HVPA):        true,
+				string(features.HVPA): true,
```

Comment:
Actually the point of this test is to test that gardenlet config is properly merged with the parent one. By removing the `ReversedVPN` usage in this test we remove the test case where feature gate is enabled in parentConfig and disabled in the gardenletConfig. Can we use another feature gate or better use dummy strings such as `FooFeature`, `BarFeature` instead of real feature names?